fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DeleteResourcePersister.java

```diff
@@ -46,13 +48,15 @@ public void persist(final OCFLPersistentStorageSession session, final ResourceOp
         final var fedoraResourceRoot = mapping.getRootObjectIdentifier();
         final var resourceId = operation.getResourceId();
         final var objectSession = session.findOrCreateSession(mapping.getOcflObjectId());
+        final var isRdf = ((DeleteResourceOperation) operation).isRdf();
         log.debug("Deleting {} from {}", resourceId, mapping.getOcflObjectId());
         if (fedoraResourceRoot.equals(resourceId)) {
             // We are at the root of the object.
             objectSession.deleteObject();
         } else {
             final var relativeSubPath = relativizeSubpath(fedoraResourceRoot, operation.getResourceId());
-            final var ocflSubPath = resolveOCFLSubpath(fedoraResourceRoot, relativeSubPath);
+            final var ocflSubPath = resolveOCFLSubpath(fedoraResourceRoot, relativeSubPath) +
```

Comment:
This seems like something that should be done within the Utils class so that it can be reused other places if needed.

Reply:
Like in the `resolveOCFLSubpath` method?

Reply:
Yeah, that would seem to make sense. I probably won't get around to it for a few weeks, but I'd like to refactor that whole pathing mess. It's very confusing.

Reply:
Except that the `resolveOCFLSubpath` really gets the subpath, which is used to resolve the sidecar files so adding `.nt` there would cause problems.

This is a difference we are dealing with in Delete because we use one persister to delete both RDFSource and NonRDFSource whereas to create or update we have two separate persisters (the RDF one adds this `.nt`).

I can duplicate the persister, but I think just having this in the DeleteResourcePersister might be ok.

Reply:
How about a `resolveRdfOcflSubpath()` method? Not a big deal either way and not a blocker for the PR.

Reply:
except that the subpath technically does not include the `.nt` we only add that here https://github.com/fcrepo4/fcrepo4/blob/cd97a2e7a869f9598b64b8fc61fa95af6db5e83b/fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtils.java#L193

I'm a little out of my depth with how this stuff is operating, if you want me to make a method I will but it will not be used for all instances of "subpath" but instead only be called to replace the above line (in `writeRDF`), [this line in `getTriples()`](https://github.com/fcrepo4/fcrepo4/blob/cd97a2e7a869f9598b64b8fc61fa95af6db5e83b/fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageSession.java#L223) and in the delete persister to refer to the exact file.

Reply:
It's fine. Just leave it.